### PR TITLE
Move to GPLv3

### PR DESCRIPTION
**Pull request by jlzaratec**
Hi 
- change LICENSE file to GPLV3
- replace text GPL 2 in variuos files
- Add licenses note in header files php and js

### CODE REVIEW

**Comment by Mte90** [comment]
I have to reject this PR for various reasons.
The ticket itself wasn't clear at all and this is my fault.
Add the license extract in all the files is not the right way in PHP and also JS and so on. This is used only in old projects and the whole project is licensed without doing it in all the files. Also adding the license on top of the php root file in wordpress create issues.
Also I see that the license is printed in all the files from the [generator](https://github.com/WPBP/generator) on the mark `{{author_license}}`.

So the only changes needed to accomplish this are (that are already part of the PR):

* Readme and Composer mentions to the license
* Change the License file

So basically you need to revert the changes to the other files. I am sorry that I didn't explained better but I forgot to do it -.-'
